community/contributing/docs/reviewing-prs.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: single
+title: Reviewing Pull Requests
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+# Review a pull request
+
+Anyone can review a documentation pull request. Visit the [pull requests](https://github.com/spinnaker/spinnaker.github.io/pulls) section in the Spinnaker website repository to see a list of all currently open pull requests.
+
+Reviewing documentation pull requests is a great way to get started contributing to the Spinnaker community. It helps you to learn the code base and builds rapport with other contributors.
+
+# Before you begin
+
+Before reviewing documentation, do the following:
+
+Read the [documentation style guide](https://www.spinnaker.io/community/contributing/docs/docs-style-guide/) so that you can leave informed feedback.
```

Comment:
This should be a relative hyperlink instead of a URL. 

`Read the [Documentation Style Guide](/community/contributing/docs/docs-style-guide/) so that you can leave informed feedback.`